cli/internal/cache/cache.go

```diff
@@ -130,8 +132,11 @@ func newSyncCache(opts Opts, repoRoot turbopath.AbsoluteSystemPath, client clien
 	}
 
 	if useHTTPCache {
+		terminal.Info(ui.Dim("• Remote computation caching enabled"))
```

Comment:
This doesn't actually solve the problem... it will still display in a `--dry-run=json` situation. Switching to use `logger` (`hclog.Logger`) instead works, but then we don't get the log by default (without `--verbosity=1` from the user. Looking at how all the other logs are displayed / hidden.